examples/basic/resources/logback.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration scan="true" scanPeriod="10 seconds">
+
+  <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
+
+  <contextName>examples.basic</contextName>
+
+  <variable name="ROOT_LOG_LEVEL" value="${ROOT_LOG_LEVEL:-info}" />
+
+  <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
We have an integration with logback. I wonder if that would be useful here. Given that it keeps breadcrumbs etc

Reply:
Yes, absolutely, but not for these examples. The purpose of the logback in these examples is to show a simple integration with an existing system (by only logging out a message, after sending an event to jetty), then also to show output on the command line.

I'll look to add in a Logback example in due course! Fantastic idea!